### ISSUE TITLE
DOC: tensordot claims to work only on 1D arrays, but works fine for 0D too

### ISSUE THREAD

**Issue by eric-wieser** [numpy/numpy#12130]
This seems pretty sensible to me, and should just have the offending claims in the docs removed:
```python
arr_0d = np.array(1)
ret = np.tensor_dot(arr_0d, arr_0d, ([], []))  # contracting no axes is well defined
assert (ret == arr_od).all()
```

More ambitiously, a test for this would be nice too if one does not already exist

**Comment by rishacha** [comment]
Can I take this up?

**Comment by eric-wieser** [comment]
Test is done, so removing that label.

Docs still say "array_like, len(shape) >= 1", which imposes unnecessary requirements on `shape`

**Comment by danielplawrence** [comment]
I've attempted to fix the documentation issue here as my first contribution:
https://github.com/numpy/numpy/pull/13071

**Comment by hameerabbasi** [comment]
Can be closed ad #13071 was merged?
